### ISSUE TITLE
shading of mountains

### ISSUE THREAD

**Issue by brunodebiasi55** [osmandapp/OsmAnd#18850]
### Describe the idea (required)

the shading of the mountains does not respect the position of the sun, it is as if we were in the southern hemisphere

### Tell us about the expected behaviour (required)

if I am in an area I know I understand where the north is and therefore I behave accordingly while in an unknown environment it is not easy to orient myself

### Tell us about alternatives you've considered (required)

I have no suggestions

**Comment by sonora** [comment]
Indeed: I have never researched why, but it seems a commonly used convention probably originating from the use of paper topo maps where North is up, is that the assumed light source for hillshading is from the top-left (which translates to the Northwest on the map). Perhaps using of "sun' approach, which would mean illuminating the map from the southerly "bottom" directions for the Northern hemisphere, would create quite an unusul experienece for experienced map users. 😉

**Comment by vshcherb** [comment]
Sun position is interesting project on its own, though I think we first will need to implement 3D buildings in order to have nice use case to watch shadows realtime

**Comment by pentamassiv** [comment]
I think it would already be helpful if the shadows of the mountains would be displayed on the correct side. In the northern hemisphere they should be in the northern side and in the southern hemisphere they should be on the southern side.

The shadows are helpful in the mountains because there can be a big difference in the amount of snow and the time it takes to melt. Where I live theres often snow until June in the shaded parts of the mountain. The snow on the other side is often gone for more than a month or two by then. Of course you could tell the shaded parts from the altitude lines, but shadows would make it a lot easier.

**Comment by vshcherb** [comment]
Just proper implementation will lead to issue with no shadows on equator, so we need to implement an option so user can select sun position which could be different by default by installation
